src/lib/components/Input.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+  export let type = 'text';
+  export let name = '';
+  export let placeholder = '';
+  export let required = false;
+  export let id = '';
+  export let value = '';
+  export let autocomplete = '';
+</script>
+
+<div>
+  <label for="first-name" class="block text-sm font-medium text-gray-700">First name</label>
```

Comment:
Shouldn't this be:

```suggestion
  <label for="first-name" class="block text-sm font-medium text-gray-700">{label}</label>
```

With a corresponding `label` prop?